src/pages/bubblegum/hashed-nft-data.md

```diff
@@ -4,4 +4,119 @@ metaTitle: Bubblegum - Hashing NFT Data
 description: Learn more about how NFT data is hashed on Bubblegum
 ---
 
-_Coming soon..._
+In previous sections we stated that each leaf node in a Bubblegum Merkle tree is obtained by hashing the data of the compressed NFT (cNFT).  But how exactly is this done?  We start with the metadata for the cNFT.  Each cNFT is minted with the following metadata structure as an argument to the minting instruction:
+
+```rust
+pub struct MetadataArgs {
+    /// The name of the asset
+    pub name: String,
+    /// The symbol for the asset
+    pub symbol: String,
+    /// URI pointing to JSON representing the asset
+    pub uri: String,
+    /// Royalty basis points that goes to creators in secondary sales (0-10000)
+    pub seller_fee_basis_points: u16,
+    // Immutable, once flipped, all sales of this metadata are considered secondary.
+    pub primary_sale_happened: bool,
+    // Whether or not the data struct is mutable, default is not
+    pub is_mutable: bool,
+    /// nonce for easy calculation of editions, if present
+    pub edition_nonce: Option<u8>,
+    /// Since we cannot easily change Metadata, we add the new DataV2 fields here at the end.
+    pub token_standard: Option<TokenStandard>,
+    /// Collection
+    pub collection: Option<Collection>,
+    /// Uses
+    pub uses: Option<Uses>,
+    pub token_program_version: TokenProgramVersion,
+    pub creators: Vec<Creator>,
+}
+```
+
+First the metadata is hashed, using the keccak256 hash function.  This hashNote that the metadata is hashed, and then hashed again with the `seller_fee_basis_points`.  This makes it easier for marketplaces to validate seller fee basis points.
```

Comment:
Typo -> "This hashNote"

Does it make sense to elaborate why it is easier?

Reply:
Added explanations.